flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/EICAS/EICAS_Common.js

```diff
@@ -13,6 +13,9 @@ class EICASCommonDisplay extends Airliners.EICASTemplateElement {
     init() {
         this.tatText = this.querySelector("#TATValue");
         this.satText = this.querySelector("#SATValue");
+        this.isaText = this.querySelector("#ISAValue");
+        this.isaContainer = this.querySelector("#ISA");
+        this.areAdirsAligned = SimVar.GetSimVarValue("L:A320_Neo_ADIRS_STATE", "Enum") == 2;
```

Comment:
also what is it with the areAdirsAligned?

Reply:
From the FCOM, I gathered that ISA is only shown when there is a valid SAT, which is when ADIRS are aligned (or I assume that's what `SimVar.GetSimVarValue("L:A320_Neo_ADIRS_STATE", "Enum") != 2` means). But since I need to check for aligned ADIRS and whether altimeter is STD, I just save the SimVar value of `refreshADIRS()` in an extra variable, so I don't need to access it again in `refreshISA()`.